### ISSUE TITLE
Markdown previews in minimap

### ISSUE THREAD

**Issue by Bellavene** [Neko-Box-Coder/MicroOmni#5]
Would be great if minimap would act as markdown previewer.

https://github.com/zyedidia/micro/issues/2994

**Comment by Neko-Box-Coder** [comment]
There's a [markdown preview](https://github.com/weebi/micro-preview) plugin if that's what you want.
The only caveat is it doesn't scroll lock with the document

Unless you are talking about wanting the minimap to work with markdown since there's no indentation for the minimap to pick up on.

**Comment by Bellavene** [comment]
Exactly, I believe the minimap could be smart and show content depending on the file type
